nemo/utils/exp_manager.py

```diff
@@ -111,6 +111,8 @@ class ExpManagerConfig:
     summary_writer_kwargs: Optional[Dict[Any, Any]] = None
     create_wandb_logger: Optional[bool] = False
     wandb_logger_kwargs: Optional[Dict[Any, Any]] = None
+    create_mlflow_logger: Optional[bool] = True
+    mlflow_logger_kwargs: Optional[Dict[Any, Any]] = None
```

Comment:
Please create a date class similar to Wandb one for default args and validation

Reply:
Just realized that you meant changing True to False...

Reply:
I meant having a dataclass with defaults that are disabled but allows users to validate their arguments in the yaml config. See wandb dataclass used here c

Reply:
Sorry I couldn't find the dataclass that you are talking about...did you mean to link something in the word `here`?

Reply:
Check how the config here -https://github.com/NVIDIA/NeMo/blob/main/nemo/utils/exp_manager.py#L72

Maps to the default config here - https://github.com/NVIDIA/NeMo/blob/main/nemo/utils/exp_manager.py#L127

Sane with ema - https://github.com/NVIDIA/NeMo/blob/main/nemo/utils/exp_manager.py#L138

It provides a default config that users should use and validate their yaml against

Reply:
I see wandb is left as Dict[any, any] which is sad, but at least the arguments are well known in our yaml files. MLFlow logger - we don't know what arguments to provide

Reply:
I see! Thanks, will change when I get time

Reply:
Sorry for delay, approved ! Looks great !